canvas_modules/common-canvas/src/common-properties/constants/constants.js

```diff
@@ -124,6 +124,8 @@ export const CONDITION_MESSAGE_TYPE = {
 
 export const SPINNER = "spinner";
 
+export const UPDATE_TYPE = "initial_load";
```

Comment:
This should be something like 
export const UPDATE_TYPE = {
  INITIAL_LOAD: "initial_load"
}

Reply:
<img width="402" alt="image" src="https://user-images.githubusercontent.com/89609176/185443928-fced22d8-6942-4535-8e63-57f3ae6e4c2a.png">

<img width="865" alt="image" src="https://user-images.githubusercontent.com/89609176/185444140-7765f596-fecb-420e-97d7-a8771a05668f.png">